src/SampleViewExtension/SampleViewExtension.csproj

```diff
@@ -112,6 +128,10 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
+    <Reference Include="VMDataBridge, Version=2.0.0.3766, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\DynamoVisualProgramming.Core.2.0.0-beta3766\lib\net45\VMDataBridge.dll</HintPath>
+      <Private>True</Private>
```

Comment:
copy local should be false - think private tag has always confused me relative to copy local.